### DISCUSSION TITLE
@mediaLibraryStyles isn't loading styles

### DISCUSSION THREAD

**Discussion by skeemer** [spatie/laravel-medialibrary#3471] [Q&A]
I tried this in Livewire v3, but it may affect v2.

The @mediaLibraryStyles directive only includes an SVG and the styles don't load in some other way.

**Comment by jules-inova** [comment]
We also have the same problem. When i look in: vendor/spatie/laravel-medialibrary-pro/src/MediaLibraryProServiceProvider.php i can see that it adds a bunch of svg icons. But what about the styling? It indeed does not seem to work.
